### ISSUE TITLE
upgrade immer?

### ISSUE THREAD

**Issue by chenyong** [mweststrate/use-immer#14]
Is there a reason `immer@1.x` is used in this module?

I noticed in our bundling that multiple versions of immer is bundled:

```
WARNING in immer
  Multiple versions of immer found:
    1.12.1 /project/~/use-immer/~/immer
    2.1.1 /project/~/@jimengio/rex/~/immer
```

**Comment by mweststrate** [comment]
Done, released as 0.2.0

**Comment by mweststrate** [comment]
Correction: 0.2.1
